### PR TITLE
Create 2018-09-10--js-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by jacobheun**
Notes for the Sep 10 js core team weekly!

- [x] Add recording link

### CODE REVIEW

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Published release candidate for `js-ipfs` 0.32.0-rc.1 and 0.32.0-rc.2
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: @alanshaw, releasing RC is an aegir step or is it done manually?

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    - Test stabilisation work [#1541](https://github.com/ipfs/js-ipfs/pull/1541)
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Nice! We should avoid the global flags though, they will hide tests that are slow when they shouldn't be.

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+- Next:
+    - Release js-ipfs 0.32
+    - Finish `--cid-base` option
+    - Create interop tests for block store get by CIDv0/1
+    - Implement block store get by CIDv0/1
+
+@achingbrain
+- Done:
+  - Spun up npm registry at https://registry.js.ipfs.io
+    - Play along at home with `npm i --registry=https://registry.js.ipfs.io`
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Wooot!! :D This is nice :D 

@achingbrain, Does it have the whole registry? What's the repo size?

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  - Fixed flaky pubsub tests https://github.com/ipfs/interop/pull/33
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: 👏🏽👏🏽👏🏽

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+   - Delegated routing support in [go-ipfs#4594](https://github.com/ipfs/go-ipfs/pull/4595).
+ - Next:
+   - libp2p-tcp transport isnt currently dialble over dns, looking into this (mafmt needs to be updated for this to work)
+   - Continue with state machine updates for libp2p connections
+
+@vasco-santos
+- Done:
+  - IPNS over Pubsub OKR
+    - Created `js-datastore-pubsub` module
+      - Initial implementation of [ipfs/js-datastore-pubsub#1](https://github.com/ipfs/js-datastore-pubsub/pull/1)
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: @vasco-santos, This design is incorrect. We don't want a datastore to publish every key that gets stored. 

Do you have a diagram of how things get plugged together? The IPNS system (module) should use PubSub to publish and subscribe to the new keys and then make the decision of storing them in the DataStore.

@vasco-santos do you have a diagram of your architecture? We can workshop this together if you need.

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      - Added routing key format [js-ipns#7](https://github.com/ipfs/js-ipns/pull/7)
+      - PRs approved, merged and released
+    - `js-ipfs`
+      - Implementation almost done 
+  - fix: libp2p records for IPNS should be signed
+    - https://github.com/ipfs/js-ipfs/pull/1543
+- Blocked:
+  - IPNS
+    - [interop#26](https://github.com/ipfs/interop/pull/26)
+      - Needs js-ipfs new release with IPNS, and to be reviewed
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Does it work locally?

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  - Create interop tests for `IPNS over Pubsub`
+  - Create tests for testing DHT interop  
+
+@hugomrdias
+- Done
+  - ipfs chunked add
+    - [#1540](https://github.com/ipfs/js-ipfs/pull/1540)
+  - ipfs-api chunked add
+    - [#851](https://github.com/ipfs/js-ipfs-api/pull/851)
+    - [stream-http/tree/fix/body-handling](https://github.com/hugomrdias/stream-http/tree/fix/body-handling)
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Does this break interop with go-ipfs?

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+  - fix `stream-http`
+    - Doesn't work inside a service-worker on Firefox [848#issuecomment-419887279](https://github.com/ipfs/js-ipfs-api/issues/848#issuecomment-419887279)
+  - add api add chunked to big data tests
+  - finish chore/improve-docs branch documentation.js for some reason doesn't find `factory-daemon`. `npx documentation serve ./js-ipfsd-ctl/src/index.js -w -f html`
+  - PR aegir to update documentation.js
+
+
+@travisperson
+ - Done:
+   - iptb [go/js/browser demo](https://asciinema.org/a/zn07AUFgZ5RWzAwit7mjpdqvW)
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: 👏🏽👏🏽👏🏽👏🏽👏🏽

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md (hunk 3 above)
Comment (achingbrain) [reply meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: At the moment it's configured to only fetch individual tarballs when they've been requested by a client and aren't present locally as the EC2 instance it's running on doesn't have enough storage to eagerly fetch the whole thing.

This issue is to sort out a more scalable storage solution, once that's in place we can hold whole the whole repo: https://github.com/ipfs/infrastructure/issues/432

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+ - Done:
+   - iptb [go/js/browser demo](https://asciinema.org/a/zn07AUFgZ5RWzAwit7mjpdqvW) 
+ - Blocked:
+ - Next:
+   - Merge iptb browser plugin
+   - Build test harness
+
+@vmx
+ - Done:
+   - Conference (FOSS4G) and travels in Tanzania
```
Comment (diasdavid) [inline meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Is there a url to the talk??

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md (hunk 3 above)
Comment (diasdavid) [reply meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Did you have the chance to download the entire registry locally to see the size it took?

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md (hunk 1 above)
Comment (alanshaw) [reply meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: Manual step at the moment until aegir supports it.

Details: https://github.com/ipfs/js-ipfs/issues/1533

PR adding step to release doc: https://github.com/ipfs/js-ipfs/pull/1536

Tracking issue in AEgir: https://github.com/ipfs/aegir/issues/261

File: meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md (hunk 9 above)
Comment (vmx) [reply meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md]: It's not easy to link to it, here's the best one I could find: https://fkih7q.attendify.io/schedule.html#9YYK6haD49GaQsORLc

**Pull request merged**